src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {person.sex === 'f' && person.isMarried ? `${person.partnerName} is my husband` : ''}
+      {person.sex === 'm' && person.isMarried ? `${person.partnerName} is my wife` : ''}
+      {person.isMarried ? '' : 'I am not married'}
```

Comment:
<img width="533" alt="image" src="https://github.com/mate-academy/react_person/assets/49764881/77f741db-a641-4d50-919e-332fb93ea8ee">


I suggest creating a variable inside your Person components for calculating this logic

```
  const partner = sex === 'm'
    ? 'wife'
    : 'husband';
    
    return (
      ...
      <p className="Person__partner">
        {
          person.isMarried
            ? `${person.partnerName} is my ${partner}`
            : 'I am not married'
        }
      </p>
    )

Reply:
hi! thank you for the clarification! I updated the code accordingly.